dockerfiles/pytorch-dev/install_rocm_from_release.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# -----------------------------------------------------------------------------
+# install_rocm_from_tarballs.sh
+#
+# Download and install ROCm tarballs for specified AMDGPU targets from
+# TheRock's nightly GitHub releases (or a forked repository).
+#
+# Usage:
+#   INSTALL_PREFIX=/pathToInstall RELEASE_VERSION=6.4.0rc20250424 ./install_rocm_from_tarballs.sh "gfx942 gfx1100"
+#
+# Environment Variables (optional):
+#   RELEASE_VERSION       - Full version string like 6.4.0rc20250424 (required if no version.json)
+#   ROCM_VERSION          - Base ROCm version like 6.4.0 (optional, auto-extracted from RELEASE_VERSION)
+#   RELEASE_TAG           - GitHub release tag to pull from (default: nightly-release)
+#   ROCM_VERSION_DATE     - Build date (default: 1 days ago)
+#   INSTALL_PREFIX        - Installation path (default: /therock/build/dist/rocm)
+#   OUTPUT_ARTIFACTS_DIR  - Directory to store downloaded tarballs (default: /rocm-tarballs)
+#   GITHUB_ORG            - GitHub organization name (default: ROCm)
+#   GITHUB_REPO           - GitHub repository name (default: TheRock)
+#
+# Requirements:
+#   curl (auto-installed if missing)
+#   jq (auto-installed if missing)
+#   bash
+#
+# Notes:
+#   - Setting GITHUB_ORG and GITHUB_REPO allows installing from forks or custom repositories.
+#   - RELEASE_VERSION controls tarball naming. ROCM_VERSION is used for internal environment setup.
+#
+# Example:
+#   GITHUB_ORG=myorg GITHUB_REPO=myrock RELEASE_VERSION=6.4.0rc20250425 ./install_rocm_from_tarballs.sh gfx942
+#
+# -----------------------------------------------------------------------------
+
+
+# Ensure required tools
+for tool in curl jq; do
+  if ! command -v "$tool" >/dev/null 2>&1; then
+    echo "[INFO] $tool not found. Installing..."
+    if command -v apt-get >/dev/null 2>&1; then
+      apt-get update
+      apt-get install -y "$tool"
+    else
+      echo "[ERROR] $tool installation not supported on this OS. Please install it manually."
+      exit 1
+    fi
+  fi
+done
+
+# Configuration
+RELEASE_TAG="${RELEASE_TAG:-nightly-release}"
```

Comment:
I am not much in favor of providing a release tag as you can derive it from the version number (`rcYYYYMMDD` refers to nightly, `dev...` to developers read). However, here Python would help with packages while you need to use a regex otherwise.

Reply:
I see - I put this since the link (_TARBALL_URL_) has in it. 
For instance:` https://github.com/ROCm/TheRock/releases/download/nightly-release/therock-dist-linux-gfx110X-dgpu-6.4.0rc20250404.tar.gz`

What would be your suggestion? Should I change its name?